File: loj/1/solution.hs
-- 思路：读取两个整数 a、b，输出 a+b。
-- 复杂度：时间 O(1)，空间 O(1)。
import Data.List

main :: IO ()
main = do
  input <- getContents
  let ws = words input
  if length ws < 2 then return () else do
    let a = read (ws !! 0) :: Integer
    let b = read (ws !! 1) :: Integer
    print (a + b)

File: loj/1/solution.pas
{ 思路：读取两个整数 a、b，输出 a+b。
  复杂度：时间 O(1)，空间 O(1)。 }
program Main;

var
  a, b: Int64;

begin
  if not eof(input) then
  begin
    read(a, b);
    write(a + b);
  end;
end.
